corehq/apps/api/es.py

```diff
@@ -559,6 +560,8 @@ def __iter__(self):
             if self.model:
                 # HACK: Sometimes the model is a class w/ a wrap method, sometimes just a function
                 if hasattr(self.model, 'wrap'):
+                    if isinstance(self.model, CommCareCase):
+                        jvalue['_source'].pop('modified_by', None)
```

Comment:
is user_id set properly, or does the value from modified_by need to go to user_id?